src/qt/sendcoinsdialog.cpp

```diff
@@ -171,15 +172,12 @@ void SendCoinsDialog::setModel(WalletModel *_model)
         ui->customFee->setSingleStep(CWallet::GetRequiredFee(1000));
         updateFeeSectionControls();
         updateMinFeeLabel();
-        updateSmartFeeLabel();
+        updateFeeLabel();
         updateGlobalFeeVariables();
 
         // set the smartfee-sliders default value (wallets default conf.target or last stored value)
         QSettings settings;
-        if (settings.value("nSmartFeeSliderPosition").toInt() == 0)
-            ui->sliderSmartFee->setValue(ui->sliderSmartFee->maximum() - model->getDefaultConfirmTarget() + 2);
-        else
-            ui->sliderSmartFee->setValue(settings.value("nSmartFeeSliderPosition").toInt());
+        ui->sliderSmartFee->setValue(settings.value("nSmartFeeSliderPosition").toInt());
```

Comment:
Can we rename this persisted key? Currently, my < 1.14.4 settings from previous installs translate to a completely different value with 1.14.5

Reply:
Done